packages/cli/package.json

```diff
@@ -67,9 +67,9 @@
   "devDependencies": {
     "@prisma/client": "workspace:*",
     "@prisma/debug": "workspace:*",
-    "@prisma/fetch-engine": "3.6.0-19.9372e13dbcc967fb9bd1f954bed16d7b7b96d4f9",
+    "@prisma/fetch-engine": "3.6.0-20.query-engine-in-filter-03d89ba07c179d4a935e06dc5b74c78a24de3164",
```

Comment:
This looks like a special engine build (integrat, that should probably not make it into `main`, right?

Reply:
Yes, once the engines PR is merged, we'll replace it with the proper build before merging this.

Reply:
https://github.com/prisma/prisma-engines/pull/2443 is merged, waiting for the binaries to be built and released

Reply:
Looks test like pipeline is stuck and this engine is not yet out from https://buildkite.com/prisma/test-prisma-engines

Reply:
@Jolg42 should be unblocked soon: https://buildkite.com/prisma/test-prisma-engines/builds/11301